src/MessageHandler/ProcessFileHandler.php

```diff
@@ -0,0 +1,166 @@
+<?php
+
+namespace App\MessageHandler;
+
+use App\Entity\File;
+use App\Message\ProcessFile;
+use App\Message\ScanFileForVirus;
+use App\Message\ZipDatasetFiles;
+use App\Repository\FileRepository;
+use App\Util\Datastore;
+use App\Util\StreamInfo;
+use Doctrine\ORM\EntityManagerInterface;
+use Psr\Log\LoggerInterface;
+use Symfony\Component\Messenger\Handler\MessageHandlerInterface;
+use Symfony\Component\Messenger\MessageBusInterface;
+
+/**
+ * The handler for file hashing messages
+ */
+class ProcessFileHandler implements MessageHandlerInterface
+{
+    /**
+     * The Entity Manager.
+     *
+     * @var EntityManagerInterface
+     */
+    private $entityManager;
+
+    /**
+     * The File Repository.
+     *
+     * @var FileRepository
+     */
+    private $fileRepository;
+
+    /**
+     * The monolog logger.
+     *
+     * @var LoggerInterface
+     */
+    private $logger;
+
+    /**
+     * Pelagos Datastore.
+     *
+     * @var Datastore
+     */
+    private $datastore;
+
+    /**
+     * Instance of symfony messenger message bus.
+     *
+     * @var MessageBusInterface
+     */
+    private $messageBus;
+
+    /**
+     * List of messages.
+     *
+     * @var array
+     */
+    private $messages = array();
+
+    /**
+     * Constructor for this Controller, to set up default services.
+     *
+     * @param EntityManagerInterface $entityManager           The entity handler.
+     * @param FileRepository         $fileRepository          The file Repository.
+     * @param LoggerInterface        $datasetFileHasherLogger Name hinted dataset_file_hasher logger.
+     * @param Datastore              $datastore               Datastore utility instance.
+     * @param MessageBusInterface    $messageBus              Symfony messenger bus interface instance.
+     */
+    public function __construct(
+        EntityManagerInterface $entityManager,
+        FileRepository $fileRepository,
+        LoggerInterface $datasetFileHasherLogger,
```

Comment:
We need to change the logger name, it does more than hashing and it shouldn't be logged in this log file